modules/commons/src/main/java/org/apache/synapse/commons/throttle/core/ConcurrentAccessController.java

```diff
@@ -49,28 +49,24 @@ public ConcurrentAccessController(int limit) {
 	 *
 	 * @return the previous value
 	 */
```

Comment:
Indentation is broken in ConcurrentAccessController file.

Reply:
Fixed in [1]

[1] https://github.com/wso2/wso2-synapse/pull/1058/commits/8f45351b62d0bcb2c1e13c3fc788890e50c32850